template-processors/base/bin/processParameters.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Kohl's Department Stores, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset
+set -o errexit
+
+echo Processing Parameters
+
+# Determine how many yaml files we have
+export YAML_COUNT="$(ls -1 $CLONED_PARAMETER_GIT_DIR/*.{yaml,yml} 2> /dev/null | wc -l)"
```

Comment:
May as well also permit `json` file extension. JSON documents are also valid YAML documents.

Reply:
Right now we only allow yaml files as the parameter inputs. We'd have to make more significant changes to all the template processor scripts to support json